config/win/BUILD.gn

```diff
@@ -282,9 +282,20 @@ config("runtime_library") {
       defines += [ "WINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP" ]
     }
     cflags_cc += [
-      "/ZW",
-      "/EHsc",
+      "/std:c++17",
+      "/Zc:__cplusplus",
     ]
+    
+    #TEMPORARY!!! Clang-cl.exe missing unwind support for Windows arm and arm64. This will generate errors in places where try/catch are used
```

Comment:
Please rephrase in a way that we can push back to google. It's okay to say this can be removed once clang-cl.exe has a feature but better to phrase properly for something we can push as I intend to push it.

Reply:
This is something that definitely shouldn't be pushed to Google. Added just to see what other issues exist for arm and arm64.

Reply:
Need it in such a way that I can submit to google. We don't want to have differences we can't submit back.